index.html

```diff
@@ -0,0 +1,216 @@
+            Safeland, our home. This document shall serve as a guide for our actions, decisions, and choices while encapsulating the values
+            we hold dear. It is our intention to live in harmony with God, our family, and the world; pursuing faith, love,
+            goodness, light, and security. In doing so, we hope to create a sanctuary of peace, happiness, and the presence of God.</p>
+
+        <h2>Article I: God</h2>
+        <ul class="principles">
+            <li>We serve only God and seek to make Him proud in all that we do, always remaining faithful to Him. We worship Him and pray to Him daily, beyond the times of meals.</li>
+            <li>We attend church each Sunday as a family and uphold the sacredness of the Sabbath.</li>
+            <li>We vow to follow the Ten Commandments, trusting in God to guide our life.</li>
+        </ul>
+
+        <h2>Article II: Family and Fellowship</h2>
+        <ul class="principles">
+            <li>We acknowledge the blessing of family. We shall appreciate and cherish our immediate and extended family, and the time we have with them.</li>
+        </ul>
+
+        <h2>Article III: Financial Security and Stability</h2>
+        <ul class="principles">
+            <li>We shall pray to obtain financial security, stability, and independence. We shall live free from financial debt
+                and manage our resources wisely.</li>
+            <li>We would like to retire safely and comfortably, without financial strain.</li>
+            <li>We would like to take one major vacation each year, in addition to zero to several smaller, domestic trips—living magical memories.</li>
+        </ul>
+
+        <h2>Article IV: Health and Well-Being</h2>
+        <ul class="principles">
+            <li>We shall hope to overflow with love for one another in our marriage.
+            </li>
+            <li>We shall attempt to eat a pure vegan diet and maintain good health, acknowledging the positive impact on our health and well-being.</li>
+            <li>We shall attempt to build happpiness in our home.</li>
+        </ul>
+
+        <h2>Article V: Work and Balance</h2>
+        <ul class="principles">
+            <li>We shall seek to live in the moment, with the spirit of God, enjoying our life and obtaining balance between work and life.</li>
+            <li>Our work will not only be a source of income but also be fulfilling to us.</li>
+            <li>We shall be very organized in our personal and work affairs.</li>
+            <li>Jacob shall attempt to obtain a Master's Degree.</li>
+            <li>Taylor shall attempt to obtain a Bachelor's Degree.</li>
+        </ul>
+
+        <h2>Article VI: Home and Family Life</h2>
+        <ul class="principles">
+            <li>We pray to live in a safe neighborhood and in a house we are pleased with.</li>
+            <li>We pray to have at least one child before Taylor reaches the age of thirty.</li>
```

Comment:
woaahhh wait a minute mr. mans. whal is this?!?!

Reply:
haha, concerns?

Reply:
yeah, kids?! ehh ahhhhh lol